### PR TITLE
Easy-Spin bug fix

### PR DESCRIPTION
**Pull request by ksrinimba**
Incorrect PV Creation script was being referred to. Also, made some minor changes to scripts (moved taint-removal from inst_oes.sh to masterinst.sh) and improved the success-messages at the end of the run.

### CODE REVIEW

File: scripts/easy-spinnaker/readme.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ c\) The application URL is
 [[http://10.168.3.10]{.underline}](http://10.168.3.10/):{DECK NodePort}
 can be obtained with this command on the master node:
 
-kubectl get svc spin-deck-ui -n oes -o
+kubectl get svc spin-deck-np -n oes -o
```
Comment (Nirmalyasen) [inline scripts/easy-spinnaker/readme.md]: The latest OES creates spin-deck-ui. Why is this being changed back to spin-deck-np?

File: scripts/easy-spinnaker/check_inst_complete.sh [hunk 2]
```diff
@@ -39,7 +39,7 @@ echo "==========================================================================
 echo 
 echo "Installation of Spinnaker is now complete. Login to the URL below using admin/OpsMx@123"
 echo 
-kubectl get svc spin-deck-ui -n oes -o jsonpath='{"http://10.168.3.10:"}{..nodePort}{"\n"}'
+kubectl get svc spin-deck-np -n oes -o jsonpath='{"http://10.168.3.10:"}{..nodePort}{"\n"}'
```
Comment (Nirmalyasen) [inline scripts/easy-spinnaker/check_inst_complete.sh]: The latest OES creates spin-deck-ui. Why is this being changed back to spin-deck-np?

File: scripts/easy-spinnaker/inst_oes.sh [hunk 3]
```diff
@@ -5,13 +5,13 @@
 #################################################################################################################
 ### PLEASE UPDATE THESE WITH THE USERNAME AND CREDENTIALS provided by OpsMX
 #################################################################################################################
-DOCKER_USERNAME=<username>
-DOCKER_PASSWORD=<password>
+DOCKER_USERNAME=USER_NAME_GOES_HERE
+DOCKER_PASSWORD=PASSWORD_GOES_HERE
```
Comment (Nirmalyasen) [inline scripts/easy-spinnaker/inst_oes.sh]: The installation will fail if the user does not update this. Do you want to add a check somewhere and fail with a proper message if the user does not update these values?

File: scripts/easy-spinnaker/readme.md (hunk 1 above)
Comment (ksrinimba) [reply scripts/easy-spinnaker/readme.md]: Fixed

File: scripts/easy-spinnaker/check_inst_complete.sh (hunk 2 above)
Comment (ksrinimba) [reply scripts/easy-spinnaker/check_inst_complete.sh]: Fixed

File: scripts/easy-spinnaker/inst_oes.sh (hunk 3 above)
Comment (ksrinimba) [reply scripts/easy-spinnaker/inst_oes.sh]: Fixed.

File: scripts/easy-spinnaker/config_hal.sh [hunk 4]
```diff
@@ -16,14 +16,13 @@
 export KUBECONFIG=/home/vagrant/.kube/config
 
 # Get nodePort numbers
-DECKNP=$(kubectl get svc spin-deck-np -n oes -o jsonpath='{...nodePort}')
+DECKNP=$(kubectl get svc spin-deck-ui -n oes -o jsonpath='{...nodePort}')
 GATENP=$(kubectl get svc spin-gate-np -n oes -o jsonpath='{...nodePort}')
```
Comment (Nirmalyasen) [inline scripts/easy-spinnaker/config_hal.sh]: Why spin-gate-np? As per https://github.com/OpsMx/enterprise-spinnaker/blob/master/charts/oes/templates/spinnaker-extra/spinsvcs.yaml it should be spin-gate-ui.

File: scripts/easy-spinnaker/readme.md (hunk 1 above)
Comment (Nirmalyasen) [reply scripts/easy-spinnaker/readme.md]: Doesn't seem it is fixed.

Comment (ksrinimba) [reply scripts/easy-spinnaker/readme.md]: Now it is.

File: scripts/easy-spinnaker/config_hal.sh (hunk 4 above)
Comment (ksrinimba) [reply scripts/easy-spinnaker/config_hal.sh]: As discussed, renamed spin-gate-np to spin-gate-ui,  changed the file name as well

**Pull request merged**